src/pages/home.tsx

```diff
@@ -1,9 +1,69 @@
-import React from "react";
+import '../components/home/home.css';
+import { ReactComponent as PageLinksIcon } from '../assets/home/arrow.svg';
+import { ReactComponent as CreateHackathonIcon } from '../assets/home/createHackathonIcon.svg';
+import { ReactComponent as HackathonAnalyticsIcon } from '../assets/home/hackathonAnalyticsIcon.svg';
+import { Link } from 'react-router-dom';
 
 export default function Home() {
-  return (
-    <div>
-      <h1>HOME</h1>
-    </div>
-  );
+	// TODO: This user name should be generic.
+	let userName: string = 'John';
+
+	const pageLinks = [
+		{ route: '/hackathon', buttonText: 'Go to Hackathon' },
+		{ route: '/attendees', buttonText: 'Attendees' },
+		{ route: '/sponsors', buttonText: 'Sponsors' },
+		{ route: '/edit-details', buttonText: 'Edit Details' },
+		{ route: '/schedule', buttonText: 'Schedule' },
+		{ route: '/settings', buttonText: 'Settings' },
+	];
+
+	return (
+		<div id="page">
+			<div id="page-header">
+				<div id="page-header-main">
+					<div id="page-header-main-generic-profile-icon" />
+					<h1 id="page-header-main-text">Welcome, {userName}.</h1>
+				</div>
+
+				<Link to="/analytics">
+					<button id="page-header-button">
+						<HackathonAnalyticsIcon />
+						<p>Hackathon Analytics</p>
+					</button>
+				</Link>
+			</div>
+
+			<h1 className="page-subheader-text" id="page-subheader-1-text">
+				Upcoming Hackathon
+			</h1>
+
+			<div id="page-links">
+				<h1 id="page-links-header">Fall 2022</h1>
+
+				<div id="page-links-grid">
```

Comment:
I think it would be a good idea to center links within the grid so that at smaller sizes (and larger sizes between the points where we cut/grow the number of links per line) we don't have so much random space on the right side like shown below.

<img width="788" alt="image" src="https://user-images.githubusercontent.com/55410292/190929746-7a681ca4-02f7-4653-b4c5-8bb16a8712b3.png">